src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/AtomicArrayWithCAS2LiveLockTest.kt

```diff
@@ -17,12 +17,14 @@ import org.jetbrains.kotlinx.lincheck.annotations.Operation
 import org.jetbrains.kotlinx.lincheck.checkImpl
 import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.ModelCheckingOptions
 import org.jetbrains.kotlinx.lincheck_test.util.checkLincheckOutput
+import org.jetbrains.kotlinx.lincheck_test.util.IgnoreInTraceDebuggerMode
 import org.junit.Test
 
 /**
  * Checks recursive spin lock representation when execution hung due to alternation of two threads
  * in recursive live-lock.
  */
+@IgnoreInTraceDebuggerMode
```

Comment:
Can we not ignore but have a different output file when `inTraceDebuggerMode == true`?

Reply:
This is one of spin-loop tests. 
It does not work when `LoopDetector` is disabled.